src/guide/single-file-component.md

```diff
@@ -65,3 +65,107 @@ After you've taken a day to dive into these resources, we recommend checking out
 The CLI takes care of most of the tooling configurations for you, but also allows fine-grained customization through its own [config options](https://cli.vuejs.org/config/).
 
 In case you prefer setting up your own build setup from scratch, you will need to manually configure webpack with [vue-loader](https://vue-loader.vuejs.org). To learn more about webpack itself, check out [their official docs](https://webpack.js.org/configuration/) and [webpack learning academy](https://webpack.academy/p/the-core-concepts).
+
+### Building with rollup
```

Comment:
I'll put this comment here but it applies more generally to this section.

From a quick look at the Rollup documentation it would appear that they use a capital R on Rollup when referring to their library. The usual convention in the Vue documentation is to adhere to the casing conventions used by a project's own documentation. So, for example, both npm and webpack are typically written in lowercase.

The `rollup` package and `rollup` command are both lowercase, so when referring to them we obviously wouldn't use a capital R. However, both of those uses would usually be surrounded with backticks when included in the middle of a sentence.

Reply:
@skirtles-code thanks! I think I've addressed this feedback along the whole piece!